runConfig.json5

```diff
@@ -0,0 +1,4 @@
+{ // Run the main (CI) pipeline configuration
+    loadDirectory: "%covid-assessment-centres/pipelines",
+    execPipeline: ["fluid.pipelines.WeCount-ODC", "fluid.pipelines.WeCount-ODC-synthetic"]
```

Comment:
The path name probably should use `execMergedPipeline` now. Same for `runUserConfig.json5` file.

Reply:
This is fixed, as well as a dependency update to the released forgiving-data - ready for another look